hack/run-ci-e2e-test.sh

```diff
@@ -24,14 +24,15 @@ set -o pipefail
 
 RUNNING_IN_CI=${JOB_NAME:-""}
 BUILD_ID=${BUILD_ID:-"${USER}-local"}
-PROVIDER=${PROVIDER:-"aws"}
-CONTAINER_RUNTIME=${CONTAINER_RUNTIME:-"docker"}
+PROVIDER=${PROVIDER:-"PROVIDER-MISSING"}
+CONTAINER_RUNTIME=${CONTAINER_RUNTIME:-}
 KUBETESTS_ROOT=$(realpath "${KUBETESTS_ROOT:-"/opt/kube-test"}")
-TEST_SET=${TEST_SET:-"conformance"}
-TEST_CLUSTER_TARGET_VERSION=${TEST_CLUSTER_TARGET_VERSION:-""}
-TEST_CLUSTER_INITIAL_VERSION=${TEST_CLUSTER_INITIAL_VERSION:-""}
-TEST_OS_CONTROL_PLANE=${TEST_OS_CONTROL_PLANE:-""}
-TEST_OS_WORKERS=${TEST_OS_WORKERS:-""}
+KUBEONE_TEST_RUN=${KUBEONE_TEST_RUN:-}
+TEST_CLUSTER_TARGET_VERSION=${TEST_CLUSTER_TARGET_VERSION:-}
+TEST_CLUSTER_INITIAL_VERSION=${TEST_CLUSTER_INITIAL_VERSION:-}
+TEST_OS_CONTROL_PLANE=${TEST_OS_CONTROL_PLANE:-}
+TEST_OS_WORKERS=${TEST_OS_WORKERS:-}
+TEST_TIMEOUT="60m"
```

Comment:
Since it's supposed to be possible to override this value (looking from `.prow.yaml`), should we use something like this instead:

```suggestion
TEST_TIMEOUT=${TEST_TIMEOUT:-"60m"}
```

Reply:
yeah.

P.S.
but did you noticed how upgrades tests finished in 60m? ;)

Reply:
Yeah, they are very consistently under 35m. This was mostly just a tiny nitpick. :slightly_smiling_face: